### PR TITLE
Homework7: Lazar Maxim

### PR DESCRIPTION
**Pull request by Wilreyyt**

### CODE REVIEW

File: homework_7/cows_and_bulls.py
Comment (Stml89) [inline homework_7/cows_and_bulls.py:1]: Everything in uppercase? Pls, use letters in the lowercase

File: homework_7/cows_and_bulls.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+while REPEAT == 1:
+    RANDOM_NUMBER = str(random.randint(1000, 9999 + 1))
+    REPEAT = 0
+    for character in RANDOM_NUMBER:
+        if RANDOM_NUMBER.count(character) > 1:
+            REPEAT = 1
+INPUT_NUMBER = ""
+while INPUT_NUMBER != RANDOM_NUMBER:
+    INPUT_NUMBER = input("Введите ваше 4-значное число: ")
```
Comment (Stml89) [inline homework_7/cows_and_bulls.py]: user still can enter non unique number

File: homework_7/statues.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"""Statues"""
+
+import random
+STATUES = []
+for i in range(4):
+    STATUE = 0
+    while STATUE == 0 or STATUE in STATUES:
+        STATUE = random.randint(1, 10)
+    STATUES.append(STATUE)
+STATUES.sort()
```
Comment (Stml89) [inline homework_7/statues.py]: you do not need to sort, the easiest way is to get min() and max() values and iterate from min to max

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Wilreyyt** [comment]
It's done i guess :)

**Review by Stml89 — APPROVED** [review]

**Pull request merged**